### PR TITLE
oc login needs to take username parameter to force basic auth

### PR DESCRIPTION
**Pull request by guillaumerose**
Related to https://github.com/code-ready/crc/issues/2032

### CODE REVIEW

File: cmd/crc/cmd/start.go [hunk 1]
```diff
@@ -211,7 +211,7 @@ Log in as user:
 
 Use the 'oc' command line interface:
   {{ .CommandLinePrefix }} {{ .EvalCommandLine }}
-  {{ .CommandLinePrefix }} oc login {{ .ClusterConfig.URL }}
+  {{ .CommandLinePrefix }} oc login {{ .ClusterConfig.URL }} --username {{ .ClusterConfig.DeveloperCredentials.Username }}
```
Comment (praveenkumar) [inline cmd/crc/cmd/start.go:214]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

File: cmd/crc/cmd/start_test.go [hunk 2]
```diff
@@ -105,7 +105,7 @@ Log in as user:
 
 Use the 'oc' command line interface:
   $ eval $(crc oc-env)
-  $ oc login https://api.crc.testing:6443
+  $ oc login https://api.crc.testing:6443 --username developer
```
Comment (cfergeau) [inline cmd/crc/cmd/start_test.go:108]: The documentation uses:
```
oc login -u developer https://api.crc.testing:6443
```
we should use that here as well.

**Review by cfergeau — CHANGES_REQUESTED** [review]
Can you c&p the problem description in the commit log?

File: cmd/crc/cmd/start_test.go (hunk 2 above)
Comment (guillaumerose) [reply cmd/crc/cmd/start_test.go]: done + commit log

**Review by cfergeau — APPROVED** [review]

**Pull request merged**